.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -30,6 +32,13 @@ import TableCard from '../../recipes/TableCard';
 
 import EmptyImage from '../../static/hand-pencil.svg';
 
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+}
+
```

Comment:
You don't need to add the `className` props because these are "pages"; they won't be reused with people passing in styles. That pattern of having a `className` prop only applies to components that are used in various contexts and may need to be customized.